docs/platforms/dotnet/common/configuration/heap-dumps.mdx

```diff
@@ -0,0 +1,79 @@
+---
+title: Heap Dumps
+sidebar_order: 23
+description: "This experimental feature allows you to automatically capture heap dumps for your application."
+---
+
+<Alert>
+
+The SDK relies on `dotnet-gcdump` to capture the heap dumps. `dotnet-gcdump` requires .NET 6 or later and must be [installed globally](https://learn.microsoft.com/en-us/dotnet/core/diagnostics/dotnet-gcdump#install) on the machine or container where the heap dumps will be captured.
+
+</Alert>
+
+Using the .NET SDK you can automatically captures heap dumps based on certain triggers. When triggered, the SDK captures the heap dump and sends an event to Sentry with the heap dump as an attachment.
+
+The basic configuration for enabling automatic heap dumps is:
+
+```csharp
+SentrySdk.Init(options => {
+    // All your other options like DSN etc.
+    ...
+    options.EnableHeapDumps(
```

Comment:
Since we  have Init and UseSentry etc depending on framework.

```suggestion
options => {
    // All your other options like DSN etc.
    options.EnableHeapDumps(
```